ldaptor/__init__.py

```diff
@@ -1,3 +1,4 @@
+"""A Pure-Python Twisted library for LDAP"""
 __version__ = "19.1.0"
```

Comment:
I didn't do the version change now - I can make the version change as a new PR

Reply:
I think that as part of this PR we should update to 20.0.0 and do a release just to make sure that it works.

This can be done as 20.0.0.rc1 release which will be triggered by a push (disable tag filtering)